src/lib/ndpi_memory.c

```diff
@@ -64,7 +64,7 @@ void *ndpi_malloc(size_t size) {
 
 void *ndpi_calloc(unsigned long count, size_t size) {
   size_t len = count * size;
-  void *p = ndpi_malloc(len);
+  void *p = _ndpi_malloc ? _ndpi_malloc(len) : malloc(len);
 
   if(p) {
     memset(p, 0, len);
```

Comment:
`len` instead of `size` on the line below?

Reply:
true..